README.md

```diff
@@ -8,7 +8,11 @@ If you encounter an issue related to these samples please [submit a new issue](h
 ## How to run the JavaScript module sample (Windows 10/Ubuntu 14+/Debian/Raspbian)
 ### Prerequisites
 1. Install latest [Git Client](https://git-scm.com/downloads).
-2. Install latest [Node LTS](https://nodejs.org).
+2. Install latest [Node Current](https://nodejs.org) (v8.4.0).
```

Comment:
Would you elaborate why we want to change from LTS to "Current"? Normally LTS is the recommended version. "Current" is for user who wants to try bleeding edge features.

Reply:
Based on the offline discussion, let's use LTS version to keep consistent with Happy Path tutorials and other Node.js related documents. Would you test if LTS works?

Reply:
Agreed, I just verified that the LTS version works so I will change it back.